packages/cli/test/integration.js

```diff
@@ -3277,6 +3277,35 @@ test('deploy gatsby twice and print cached directories', async t => {
   }
 });
 
+test('deploy pnpm twice using pnp and symlink=false', async t => {
+  const directory = path.join(__dirname, 'fixtures/unit/pnpm-pnp-symlink');
+  console.log(directory);
+  function deploy() {
+    return execa(binaryPath, [...defaultArgs, '--public', '--confirm'], {
+      directory,
+      stdio: 'inherit',
+      reject: true,
+    });
+  }
+
+  function logs(deploymentUrl) {
+    return execa(binaryPath, ['logs', deploymentUrl], {
+      stdio: 'inherit',
+      reject: true,
+    });
+  }
+
+  let { stdout: deploymentUrl } = await deploy();
```

Comment:
@styfle this deployment isn't working correctly. Do you know what I'm doing wrong here?

Reply:
figure out it was an issue with the execa configuration